customer-satisfaction/steps/model_loader.py

```diff
@@ -0,0 +1,20 @@
+from sklearn.base import RegressorMixin
+from zenml import step, ModelVersion
+from zenml.client import Client
+
+
+@step
+def model_loader(
+    model_name: str
+) -> RegressorMixin:
+    """Implements a simple model loader that loads the current production model.
+
+    Args:
+        model_name: Name of the Model to load
+    """
+    model_version = ModelVersion(
```

Comment:
this should use the step_context to get the model version

Reply:
The problem with that is, that I then get the model that was trained by the current pipeline, not the model that si in the production stage. At least thats my best understanding of this all.

Reply:
Why do you need this loader step at all actually?

Reply:
because the continuous_..._pipeline first trains a model, then optionally promotes it, and then the model_deployer needs to have the "Production" model. This might be a different model from the recently trained model. The StepContext only allows me to access the recently trained model though as this is all happening in the same pipeline context.

I also can not load the ModelVersion directly in the pipeline without the step, as i need to make sure this runs "after" the training pipeline. I have not found a way to do this without wrapping this in a step.